packages/trader/src/Modules/MT5/Components/mt5-account-card.jsx

```diff
@@ -35,108 +36,111 @@ const MT5AccountCard = ({
     onClickFund,
     onPasswordManager,
 }) => {
-    const IconComponent = icon || (() => null);
-    const lbl_add_account =
-        type.category === 'real' ? (
-            <Localize i18n_default_text='Add real account' />
-        ) : (
-            <Localize i18n_default_text='Add demo account' />
-        );
+    const lbl_add_account = <Localize i18n_default_text='Add account' />;
     const cta_label = button_label || lbl_add_account;
-
-    const has_popular_banner = type.type === 'synthetic' && type.category === 'real';
     const has_demo_banner = type.category === 'demo';
 
-    return (
-        <div className={classNames('mt5-account-card', { 'mt5-account-card__logged-out': !is_logged_in })}>
-            {has_popular_banner && (
-                <div className='mt5-account-card__banner'>
-                    <Localize i18n_default_text='Most popular' />
-                </div>
-            )}
-            {has_demo_banner && (
-                <div className='mt5-account-card__banner mt5-account-card__banner--demo'>
-                    <Localize i18n_default_text='DEMO' />
+    const renderHeader = () => {
+        return (
+            <React.Fragment>
+                <div
+                    className={classNames('mt5-account-card__type', {
+                        'mt5-account-card__type': has_demo_banner,
+                    })}
+                    id={`mt5_${type.category}_${type.type}`}
```

Comment:
what is this id needed for?

Reply:
Code does not belong to me and im guessing its related to test automations.

Reply:
All code belongs to you as you are one of us now 😅 afaik there are currently no test-automations for deriv

Reply:
hahah true that but i spoke to @negarn and she said to keep the old code as it is 😄